examples/front-proxy/service.py

```diff
@@ -1,9 +1,23 @@
 from flask import Flask
+from flask import request
 import socket
 import os
+import sys
+import requests
 
 app = Flask(__name__)
 
+TRACE_HEADERS_TO_PROPAGATE = [
+    'X-Client-Trace-Id',
```

Comment:
that's not required to propagate, essentially it's only used on the first hop so that client traces can be merged with the backend traces